jina/drivers/rank.py

```diff
@@ -28,11 +29,12 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.recursion_order = 'post'
 
-    def _apply_all(self, docs: 'jina_pb2.Document', context_doc: 'jina_pb2.Document', *args, **kwargs):
+    def _apply_all(self, docs: List['jina_pb2.Document'], context_doc: 'jina_pb2.Document', travers_on: str, *args, **kwargs):
```

Comment:
if you don't need `traverse_on`, no need to add it. That's how *args and **kwargs for